src/wired/error_boundry.tsx

```diff
@@ -1,32 +1,65 @@
 import Error, { ErrorProps } from 'next/error';
 import React, { Component, PropsWithChildren } from 'react';
+import { NextPageContext } from 'next';
 
 export type WiredErrorBoundaryProps = PropsWithChildren<{
-  ErrorComponent?: React.ComponentType<ErrorProps>;
+  ErrorComponent?: React.ComponentType<
+    ErrorProps & { error: NextPageContext['err'] }
+  >;
 }>;
 
 interface WiredErrorBoundaryState {
   hasError: boolean;
+  error: NextPageContext['err'];
 }
 
 export class WiredErrorBoundary extends Component<
   WiredErrorBoundaryProps,
   WiredErrorBoundaryState
 > {
-  static getDerivedStateFromError(): WiredErrorBoundaryState {
-    return { hasError: true };
+  static getDerivedStateFromError(
+    error: NextPageContext['err']
+  ): WiredErrorBoundaryState {
+    return { hasError: true, error };
   }
 
-  state = { hasError: false };
+  state = {
+    hasError: false,
+    error: {
```

Comment:
Can you combine `hasError` and `error` states to just a nullable `error`?

Reply:
Sure, will have that done shortly.

Reply:
@ederski implemented feedback 🤙🏻